Apsitvarkom.DataAccess/PollutedLocationDTOFileRepository.cs

```diff
@@ -25,27 +25,34 @@ private PollutedLocationsDTOFileRepository(Stream stream)
     }
 
     /// <inheritdoc />
-    public async Task<IEnumerable<PollutedLocationDTO>> GetAllPollutedLocationsAsync()
+    public async Task<IEnumerable<PollutedLocationDTO>> GetAllAsync()
     {
         var result = await JsonSerializer.DeserializeAsync<IEnumerable<PollutedLocationDTO>>(_stream, _options);
         return result ?? Enumerable.Empty<PollutedLocationDTO>();
     }
 
+    /// <inheritdoc />
+    public async Task<PollutedLocationDTO?> GetByIdAsync(string id)
+    {
+        var result = await GetAllAsync();
```

Comment:
NIT: `result` is a bit generic here, maybe a better name would be `allLocations` or something more specific?